### PR TITLE
Replaces a window with a wall

### PR DESCRIPTION
**Pull request by tf-4**
## About The Pull Request

Replaces one window with one wall

image

## Why It's Good For The Game

No more shooting through the window when testing guns in the range

## Changelog
:cl:
fix: Replaced the rear window of the firing range with a wall
/:cl:

### CODE REVIEW

**Comment by RatFromTheJungle** [comment]
cring not cool and very unepoic!!!

File: _maps/map_files/Deltastation/DeltaStation2_skyrat.dmm [hunk 1]
```diff
@@ -168635,7 +168635,7 @@ bnI
 ciP
 bgZ
 clA
-cns
+clA
```
Comment (RatFromTheJungle) [inline _maps/map_files/Deltastation/DeltaStation2_skyrat.dmm:168638]: how did the change do it only one lettert

**Review by RatFromTheJungle — APPROVED** [review]
how did this do the change it onaly two leter plese explan

File: _maps/map_files/Deltastation/DeltaStation2_skyrat.dmm (hunk 1 above)
Comment (tf-4) [reply _maps/map_files/Deltastation/DeltaStation2_skyrat.dmm:168638]: skill

**Pull request merged**